public/js/showoff.js

```diff
@@ -787,16 +787,8 @@ function blankScreen()
   }
 }
 
-var notesMode = false
-function toggleNotes()
-{
-  notesMode = !notesMode
-	if (notesMode) {
-		$('#notesInfo').show()
-		debug('notes mode on')
-	} else {
-		$('#notesInfo').hide()
-	}
+function toggleFooterNotes() {
```

Comment:
hmm. I'm not sure that the collision is a bad thing. Is it convenient to be able to pop out the notes window with a keyboard shortcut? Should we add more shortcuts to the presenter?